src/main/java/frontend/AdvancedTab.java

```diff
@@ -0,0 +1,37 @@
+package frontend;
+
+import javax.swing.*;
+import java.awt.*;
+
+/**
+ * Created by mattpatera on 3/19/17.
+ */
+class AdvancedTab extends JLabel {
```

Comment:
JPanels inside of a JLabel is concerning and confusing. It seems to work but is this structure really ideal?